.devcontainer/compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  site:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+    command: sleep infinity
+    ports:
+      - "4000"
```

Comment:
I can't seem to connect to the site through the dynamic port. Steps I did:
 - Run `docker ps` to see the dynamic port for `mobility-marketplace_devcontainer-site-1`
 - Go to `http://localhost:<dynamic port number>`

Firefox says `The connection was reset`.

I can see the site at `http://127.0.0.1:4000`, but live-reload doesn't work.

Reply:
Yeah you are right, I'm seeing the same thing... hmmm 🤔

Reply:
Actually... LiveReload just started working 🤷 On the `http://127.0.0.1:4000` URL

Reply:
Oh interesting, it started working for me too. 🤔